vertica/assets/dashboards/overview.json

```diff
@@ -1,453 +1,816 @@
+                                            "query": "avg:vertica.node.resource_requests{$service}"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "style": {
+                                        "line_type": "solid",
+                                        "line_width": "normal",
+                                        "palette": "dog_classic"
+                                    }
+                                }
+                            ],
+                            "show_legend": true,
+                            "time": {
+                                "live_span": "4h"
+                            },
+                            "title": "Resource requests",
+                            "title_align": "left",
+                            "title_size": "16",
+                            "type": "timeseries"
+                        },
+                        "id": 13,
+                        "layout": {
+                            "height": 2,
+                            "width": 6,
+                            "x": 0,
+                            "y": 4
+                        }
+                    }
+                ]
+            },
+            "id": 8233699102311070,
+            "layout": {
+                "height": 7,
+                "is_column_break": true,
+                "width": 6,
+                "x": 0,
+                "y": 0
             }
-          ],
-          "custom_links": [],
-          "title": "Resource requests",
-          "title_size": "16",
-          "title_align": "left",
-          "time": {
-            "live_span": "4h"
-          },
-          "show_legend": false,
-          "legend_size": "0"
         },
-        "layout": {
-          "x": 82,
-          "y": 46,
-          "width": 47,
-          "height": 12
-        }
-      },
-      {
-        "id": 14,
-        "definition": {
-          "type": "timeseries",
-          "requests": [
-            {
-              "q": "avg:vertica.thread.active{*} by {node_name}, avg:vertica.socket.open{*} by {node_name}",
-              "display_type": "bars",
-              "style": {
-                "palette": "dog_classic",
-                "line_type": "solid",
-                "line_width": "normal"
-              }
+        {
+            "definition": {
+                "columns": [
+                    "host",
+                    "service"
+                ],
+                "indexes": [],
+                "message_display": "expanded-md",
+                "query": "source:vertica $service",
+                "show_date_column": true,
+                "show_message_column": true,
+                "sort": {
+                    "column": "time",
+                    "order": "desc"
+                },
+                "title": "",
```

Comment:
nit - You could give a name to this widget

Reply:
Done!